shared/middleware/helpers/bucket_helper.rb

```diff
@@ -24,7 +24,7 @@ def category_from_file_type(extension)
     mime_type = Sinatra::Base.mime_type(extension)
     if mime_type == 'application/pdf'
       'pdf'
-    elsif ['.doc', '.docx'].include? extension
```

Comment:
Are you sure we want this to be `doc`? Doesn't that imply MS word format? The mime type for .txt is `text/plain`. I don't have enough context for this function.